trace/trace.go

```diff
@@ -194,7 +195,14 @@ func (t *Trace) ClientRecord(cl *Client, name string, tags map[string]string) er
 	span := t.SSFSpan()
 	span.Name = name
 
-	return Record(cl, span, t.Sent)
+	err := Record(cl, span, t.Sent)
+	if err != nil {
+		cl.statsMtx.Lock()
+		defer cl.statsMtx.Unlock()
+		tags := []string{fmt.Sprintf("error:%s", err.Error())}
+		cl.stats.Incr("veneur.trace.record.errors_total", tags, 0.1)
```

Comment:
I think the metrics prefix that we set on our statsd client will cause the metric to be reported as `veneur.veneur.trace.<...>`. I wonder if `SetErrorStats` should have a sister method to set the metric name, too.